build.psm1

```diff
@@ -539,6 +539,12 @@ Fix steps:
         }
     }
 
+    if ($Environment.IsUbuntu)
+    {
+        # psrp 1.3* now depends on libgssapi_krb5 for Kerberos support.
+        $null = New-Item -Force -ItemType SymbolicLink -Target "/usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2" -Path "$publishPath/libgssapi_krb5.so" -ErrorAction Stop
```

Comment:
The two symbolic links about `/lib64/libssl.so.10` are also created in `packaging.psm1`, see https://github.com/PowerShell/PowerShell/blob/master/tools/packaging/packaging.psm1#L466

Do you need to create this one there too? It seems to me we don't need to create in packaging.psm1, but not sure on that.

Reply:
I don't know why their being created in packaging.psm1.  My understanding is creating them at build time in the publish directory ensures they are in the package and will be created at install time.

Reply:
Maybe the two symbolic links about `/lib64/libssl.so.10` created in `packaging.psm1` should be removed?
When testing Linux PSRP, can you please create a powershell package and use the package to test? We will know for sure that it works end-to-end.

Reply:
It turns out that the nuget package does have the symbolic link but the deb package did not. In the process of creating the staging directory, the actual file was copied. I've updated packaging.psm1 to address this.

Reply:
> It turns out that the nuget package does have the symbolic link

Which nuget package are you talking about?

> I've updated packaging.psm1 to address this

Maybe you forget to push the commit that has this change?

Reply:
Sorry, I didn't see this comment.  You're right, I hadn't pushed the change. It's there now.

Reply:
@daxian-dbw  Do you have any addition comments?